app/lib/omniauth/strategies/true_layer.rb

```diff
@@ -26,10 +31,15 @@ def token_params
       end
 
       def enable_mock
-        env_setting = ENV['TRUE_LAYER_ENABLE_MOCK']
-        return false if env_setting.blank?
+        Rails.configuration.x.true_layer.enable_mock
+      end
 
-        ActiveModel::Type::Boolean.new.cast(env_setting)
+      # If consent_id is set, the flow will skip TrueLayer's consent page.
+      # consent_id is used by TrueLayer for tracking purposes and needs to be a static string specific to our TrueLayer account.
+      # It could be something different than true_layer.client_id but it makes sense to use this value
```

Comment:
I was a little concerned that this would expose the client id, but checking the Oauth2 documentation, I don't think this is a problem. For example, I found this quote: ["The client ID is considered public information, and is used to build login URLs"](https://aaronparecki.com/oauth-2-simplified/)